src/index.ts

```diff
@@ -524,8 +524,12 @@ async function transferMergeRequests() {
       i => i.title.trim() === mr.title.trim()
     );
     let githubIssue = githubIssues.find(
-      // allow for issues titled "Original Issue Name [merged]"
-      i => i.title.trim().includes(mr.title.trim())
+      // allow for issues titled "Original Issue Name - [merged|closed]"
+      i => {
+        // regex needs escaping in case merge request title contains special characters
+        const regex = new RegExp(escapeRegExp(mr.title.trim()) + ' - \\[(merged|closed)\\]');
```

Comment:
I would like if this wouldn't be a regex. Could you rewrite it to be something along the lines of:
```js
i.title.trim().startsWith(mr.title.trim())
```